### DISCUSSION TITLE
Curl error 35

### DISCUSSION THREAD

**Discussion by stevewillia** [Security-Onion-Solutions/securityonion#6516] [General] [answered]
Trying to reinstall 2.3.90 and in the so setup.log getting the error messages.

[WARNING] (10/10) Could not reach https://repo.saltstack.com/py3/ubuntu/18.04/amd64/archive/3003/SALTSTACK-GPG-KEY.pub, curl error code: 35

[WARNING] (01/10) Could not reach https://packages.wazuh.com/key/GPG-KEY-WAZUH, curl error code: 35

And the preflight fails.

This is not an Airgap installation.  Doing a network install

From what I've read, curl error 35 is a SSL connection issue.  There is not a proxy

Running curl

curl -v  https://packages.wazuh.com
* Rebuilt URL to: https://packages.wazuh.com/
*   Trying 13.226.204.18...
* TCP_NODELAY set
* Connected to packages.wazuh.com (13.226.204.18) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to packages.wazuh.com:443
* stopped the pause stream!
* Closing connection 0
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to packages.wazuh.com:443

Any Ideas
Steve

**Comment by presianbg** [comment]
Hi,

Looks like a geo location + cloudfront issue or maybe DNS. 
From my system `packages.wazuh.com` is being resolved on completely different set of IPs:

```
packages.wazuh.com has address 52.85.7.126
packages.wazuh.com has address 52.85.7.39
packages.wazuh.com has address 52.85.7.118
packages.wazuh.com has address 52.85.7.108
```

Can you try with different DNS server ?

Cheers,
PY

**Reply by stevewillia** [reply]
I changed the dns to use google 8.8.8.8 and same results.

Thank you.  I'll keep looking

**Comment by defensivedepth** [comment, marked as answer]
Looks like something in-between the SO server and the destination is closing the connection. 

Can you confirm that there are no firewalls or content filtering etc that might be blocking the request?

**Comment by stevewillia** [comment]
Turning off content filer as I write this

Thanks
